src/main/java/io/confluent/connect/hdfs/HdfsSinkConnector.java

```diff
@@ -69,4 +73,20 @@ public void start(Map<String, String> props) throws ConnectException {
   public void stop() throws ConnectException {
 
   }
+
+  @Override
+  protected ConfigDef defineConfig() {
+    return HdfsSinkConnectorConfig.getConfig();
```

Comment:
Are we always going to have a class of `AbstractConfig`? Should we think about making that the type that's returned rather than `ConfigDef`. Not really sure which is better, this just looks like in practice it might always be implemented as `return SubclassOfAbstractConfig.getConfig();`.